### DISCUSSION TITLE
[Feature] Passkey support

### DISCUSSION THREAD

**Discussion by eiqnepm** [immich-app/immich#11364] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

I understand that Immich decided to rely on third party solutions for 2FA due to there being so many 2FA standards, I should imagine not having to implement and support various 2FA standards makes development less of a headache.

Please may Immich add first party support for passkeys? Passkeys are designed to replace all current authentication methods, pretty much the be all and end all of authentication, as they replace both traditional passwords and 2FA. Passkeys are widely adopted by most if not all web browsers, and operating systems.

Having a first party authentication method that is equally, if not more secure than the traditional password plus 2FA is a huge benefit to those of us who either don't know how or are unwilling to manage their own third party 2FA solution.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
#3338

**Reply by eiqnepm** [reply]
> #3338

Thanks! I searched for through the issues and selected new issue which created this discussion, I wasn't aware feature requests were done via discussions.

**Comment by Drun555** [comment]
> are unwilling to manage their own third party 2FA solution

I can only recomend to try Pocket ID. It's incredible simple to setup and looks gorgeous. If you managed to host Immich, Pocket ID will be a piece of cake.
